core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockStoreMeta.java

```diff
@@ -84,9 +84,7 @@ public long getCapacityBytes() {
     Map<String, List<String>> pathsOnTiers = new HashMap<>();
     for (Pair<String, String> tierPath : mCapacityBytesOnDirs.keySet()) {
       String tier = tierPath.getFirst();
-      if (pathsOnTiers.get(tier) == null) {
-        pathsOnTiers.put(tier, new ArrayList<String>());
-      }
+      pathsOnTiers.computeIfAbsent(tier, k -> new ArrayList<>());
```

Comment:
why not `xxx.computeIfAbsent(xxx).add(xxx)`, no reason to `get(tier)` again in the line below.

Reply:
Thanks for your advice.